src/config/networks.ts

```diff
@@ -66,102 +59,52 @@ export const NETWORKS: Networks = {
       stakeTarget: 0.75,
     },
   },
-  kusama: {
-    name: 'Kusama',
+  alephzerotestnet: {
```

Comment:
Here starts the main config for the network. There's some colors defined here, but also rpc addresses, ss58, unitsetc.